README.windows.md

```diff
@@ -227,85 +278,110 @@ Vagrant.configure("2") do |config|
-This is typically quite a bit of work, so we will use [this script](http://sourceforge.net/projects/mingw-w64-dgn/) o make it easy.
-
-1. `svn checkout svn://svn.code.sf.net/p/mingw-w64-dgn/code/trunk mingw-w64-dgn`
-2. `cd mingw-w64-dgn`
-3. edit `rebuild_cross.sh` and make the following two changes:
-  a. uncomment `export MAKE_OPT="-j 2"`, if appropriate for your machine
-  b. add `fortran` to the end of `--enable-languages=c,c++,objc,obj-c++`
-5. `bash update_source.sh`
-4. `bash rebuild_cross.sh`
-5. `mv cross ~/cross-w64`
-6. `export PATH=$HOME/cross-w64/bin:$PATH` # NOTE: it is important that you remember to always do this before using make in the following steps! You can put this line in your .profile to make it easy.
+### Cross-building Julia without Vagrant
 
-Then we can essentially just repeat these steps for the 32-bit compiler, reusing some of the work:
+If you don't care that the build is potentially incompatible with the WinRPM
+ecosystem (or happen to be on opensuse), use the following steps to cross-
+compile julia:
 
-7. `cd ..`
-8. `cp -a mingw-w64-dgn mingw-w32-dgn`
-9. `cd mingw-w32-dgn`
-10. `rm -r cross build`
-11. `bash rebuild_cross.sh 32r`
-12. `mv cross ~/cross-w32`
-13. `export PATH=$HOME/cross-w32/bin:$PATH` # NOTE: it is important that you remember to always do this before using make in the following steps! You can put this line in your .profile to make it easy.
+First, you will need to ensure your system has the required dependencies.  We
+need wine (>=1.7.5), a system compiler, and some downloaders.
 
-Note: for systems that support rpm-based package managers, the necessary dependencies can be downloaded from the OpenSUSE build service (see the Vagrant script above).
+**On Ubuntu** (on other linux systems, the dependency names are likely to be similar):
+```sh
+apt-add-repository ppa:ubuntu-wine/ppa
+apt-get update
+apt-get install wine1.7 subversion cvs gcc wget p7zip-full
+```
 
-@vtjnash occassionally upload his builds, so you can also download those to save build time:
-- [x86_64-w64-mingw64 gcc-5.2.0 MacOS-10.10 (105 MB)](https://onedrive.live.com/redir?resid=BCAF288A35FC4406!1601&authkey=!ANpFOoCqFGYTcHM&ithint=file%2cgz)
-- [i686-w64-mingw64 gcc-5.2.0 MacOS-10.10 (89 MB)](https://onedrive.live.com/redir?resid=BCAF288A35FC4406!1607&authkey=!AFrL8G5G70iYhUU&ithint=file%2cgz)
-- [*-w64-mingw64 gcc-4.9.2 Ubuntu-15.04 (168 MB)](https://onedrive.live.com/redir?resid=BCAF288A35FC4406!1602&authkey=!ABhstrqDG-4zdLo&ithint=file%2cgz)
+**On Mac**: Install XCode, XCode command line tools, X11 (now [XQuartz](
+http://xquartz.macosforge.org/)), and [MacPorts](http://www.macports.org/install.php)
+or [Homebrew](http://mxcl.github.io/homebrew/).  Then run `port install wine wget`,
+or `brew install wine wget`, as appropriate.
+
+**On Both:**
+Unfortunately, the version of gcc installed by Ubuntu targets pthreads.  On Mac,
+the situation is similar: the version in MacPorts is very old and Homebrew does
```

Comment:
These statements (and the rest of the section) are false now. Macports and Homebrew have `mingw-w64` now, and our build now requires the pthreads thread model instead of win32.